ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh

```diff
@@ -121,6 +121,10 @@ aws|aws-arm64)
     export KUBE_SSH_USER=core
     ;;
 azure4) export TEST_PROVIDER=azure;;
+azurestack) 
+    export TEST_PROVIDER=azurestack
+    export AZURE_AUTH_LOCATION=${SHARED_DIR}/osServicePrincipal.json
+    ;;
```

Comment:
Is this a guess or did you have a chance to look into it? My guess would be to just use the Azure tests:
```suggestion
azure4|azurestack) export TEST_PROVIDER=azure;;
```
There is no need to use auth at this point. 

My thought is to just use the azure test initially and we can improve this to figure out if there is azurestack-specific testing (unless you have already done that).

Reply:
did you ever try this suggestion of `TEST_PROVIDER=azure`?